cli/cmd/project/rename.go

```diff
@@ -58,11 +58,9 @@ func RenameCmd(cfg *config.Config) *cobra.Command {
 				return err
 			}
 
-			cmdutil.PrintlnSuccess("Renamed project")
-			cmdutil.PrintlnSuccess(fmt.Sprintf("New web url is: %s\n", updatedProj.Project.FrontendUrl))
-			cmdutil.TablePrinter(toRow(updatedProj.Project))
-
-			return nil
+			ch.Printer.PrintlnSuccess("Renamed project")
+			ch.Printer.PrintlnSuccess(fmt.Sprintf("New web url is: %s\n", updatedProj.Project.FrontendUrl))
```

Comment:
Can remove extra `Sprintf`. Also applies to many other places.

Reply:
Keeping as is for now as `PrintlnSuccess ` requires string as args